lint.go

```diff
@@ -1444,6 +1445,24 @@ func (f *file) lintContextArgs() {
 	})
 }
 
+// lintGoGenerateDirective lints go:generate directives
+func (f *file) lintGoGenerateDirective() {
+	for _, cg := range f.f.Comments {
+		for _, c := range cg.List {
+			txt := c.Text
+			if !strings.HasPrefix(txt, "//") {
+				continue
+			}
+			txt = txt[2:]
+			// if there's space before go:generate, this was probably a mistake
+			if !strings.HasPrefix(txt, "go:generate") &&
```

Comment:
There are [more pragmas](https://golang.org/pkg/cmd/compile/internal/gc/#Pragma), not just `go:generate`. You may want to check for  `// go:`.